apps/help-center/help-center-gutenberg.js

```diff
@@ -61,9 +61,9 @@ function HelpCenterContent() {
 				onClick={ handleToggleHelpCenter }
 				icon={ <HelpIcon /> }
 				label="Help"
-				aria-pressed={ canvasMode === 'edit' && show ? true : false }
+				aria-pressed={ ( ! canvasMode || canvasMode === 'edit' ) && show ? true : false }
```

Comment:
Is this the equivalent of 

```suggestion
				aria-pressed={ canvasMode === 'edit'  ||  show }
```

Reply:
No. For example, if the `canvasMode` is `default` or `view`, the result would be false even if the show is true